packignore

```diff
@@ -16,3 +16,4 @@ icons\icon_256.png
 Cmder.bat
 vendor\tmp
 appveyor.yml
+vendor\cmder.sh
```

Comment:
Yes.  The build.ps1 puts it where it needs to be whenrunning a bash shell.  
I ignore it in packing to avoid confusion of having the same file in two 
places, vendor/cmder.sh is not used when running cmder.


On November 10, 2015 3:20:06 AM Martin Kemp <notifications@github.com> wrote:

>> @@ -16,3 +16,4 @@ icons\icon_256.png
>>  Cmder.bat
>>  vendor\tmp
>>  appveyor.yml
>> +vendor\cmder.sh
>
> Are you sure this should be ignored?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cmderdev/cmder/pull/702/files#r44384444